pkgs/os-specific/linux/keyutils/default.nix

```
@@ -21,7 +21,7 @@ stdenv.mkDerivation rec {
       sha256 = "0wnvbjfrbk7rghd032z684l7vk7mhy3bd41zvhkrhgp3cd5id0bm";
     })
     ./conf-symlink.patch
-  ];
+  ] ++ lib.optionals (stdenv.cc.isClang) [ ./0001-Remove-unused-function-after_eq.patch ];
```

```suggestion
    ./0001-Remove-unused-function-after_eq.patch
  ];
```

Does anything break if we apply this all the time?

---

I would not want to make make that call.
I can say that the patched package builds fine using both `gcc` and `clang` but that is about all the testing i have done `gcc` wise.
I never heard anything back from upstream either with regards to the patch I sent.
I'd be plus 0 on it, but if we decide to remove the `clang` switch I would include a comment explaining why the patch is needed, as a normal build using the standard `gcc` stdenv would be fine either way.

---

It would be nice if we could do that. When we update the package it is unlikely that some explicitly tests it with clang and the patch would bit rot.

---

Done